src/DeviceRendererFactory.js

```diff
@@ -203,24 +209,33 @@ module.exports = class DeviceRendererFactory {
      * @param  {Object}             options  Various configuration options.
      */
     addPlugins(instance, options) {
+        /*
+         * Load instance dedicated plugins
+        * If addCustomPlugins is called after new plugin.class we get an error for option.camera cause Camera rendertoolbar link mediaManager toggleview which is not yet defined (cause defined in addCustomePlugins)
```

Comment:
I think another way to fix this is what I mentioned here: https://github.com/Genymobile/genymotion-device-web-player/pull/47#discussion_r1444381324 fixing my own mistake and making the MediaManager a default plugin and not a custom one.

Reply:
I know this is a very bad fix, i have made a quick change to avoid error in console, but i think we need a little rework on the loading workflow. For me it's another story